src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -1,5 +1,6 @@
 import BaseSimidCreative from '../base_simid_creative.js';
 import {CreativeMessage, CreativeErrorCode} from '../constants.js';
+import UserActivityLogger from './UserActivityLogger.js';
```

Comment:
Try to keep these alphabetical, adapting the rules from here: https://google.github.io/styleguide/jsguide.html#file-goog-require
as best as possible

Reply:
do you mean the creativeMessage and creativeErrorCode?

Reply:
fixed